assignment3.java

```diff
@@ -0,0 +1,30 @@
+package assignment3;
+
+import java.util.Scanner;
+
+/**
+ * @author Adam
+ */
+public class Assignment3 {
+    
+    public static void main(String[] args) {
+        /*This program asks the user for a word. It then proceeds to reverse the
+        order of the letters in a for loop. Then it displays the ne word to the user.
+        */
+
+        int length;                 //holds word length
+        String  word,               //holds user entered string
+                letter,             //holds a letter of the word
+                wordReverse = "";   //holds the user word in reverse order
+        Scanner keyboard = new Scanner(System.in);
+       
```

Comment:
I like the way you declared your variables here, using the comma formatting to seperate each variable but still giving each variable its own line. I never thought to do it this way. I might in the future!